libs/detectors/x86/alphapose/builders/builder.py

```diff
@@ -0,0 +1,21 @@
+from models.fastpose import FastPose
```

Comment:
Add the reference if you copy this module from another repo otherwise it's not neccessary.